api/src/main/java/org/openmrs/module/ugandaemrreports/reporting/metadata/Metadata.java

```diff
@@ -38,6 +38,14 @@
         public static final String TB_TREATMENT_START_DATE = "dce02eca-30ab-102d-86b0-7a5022ba4115";
         public static final String ON_TB_TREATMENT = "dcdaa6b4-30ab-102d-86b0-7a5022ba4115";
         public static final String ASSESSED_FOR_MALNUTRITION = "dc655734-30ab-102d-86b0-7a5022ba4115";
+
+        public static final String TETANUS_DOSE_GIVEN = "39217e3d-6a39-4679-bf56-f0954a7ffdb8";
+		public static final String FIRST_DOSE = "0192ca59-b647-4f88-b07e-8fda991ba6d6";
```

Comment:
@ssmusoke, the dosage concepts are generic as defined in the concept dictionary and may be reused in the future for defining dosages for other drugs.. hence I don't think it's necessary to prefix them.